community-plugins.json

```diff
@@ -4838,5 +4838,12 @@
         "author": "Ellpeck",
         "description": "Multi-purpose time trackers for your notes!",
         "repo": "Ellpeck/ObsidianSimpleTimeTracker"
+    },
+    {
+        "id": "squiggle",
+        "name": "Squiggle",
+        "author": "Jesse Hoogland",
+        "description": "Display a list of recently opened files",
```

Comment:
I think this is the wrong description for this plugin? Do you mind changing this to be descriptive of what your plugin does, then I can merge it? Thanks!

Reply:
Yep! It should be good now. Thanks for your patience!